sdk/lib/opentelemetry/sdk/trace/export/batch_span_processor.rb

```diff
@@ -25,20 +27,23 @@ module Export
         # export fails with {FAILED_RETRYABLE}, backing off linearly in 100ms
         # increments.
         class BatchSpanProcessor
+          EXPORT_TIMEOUT_MILLIS = 30_000
```

Comment:
Transcription error, maybe (by me)? We should stick close to the spec here, so `EXPORTER_TIMEOUT_MILLIS`.

Reply:
:heavy_check_mark: